src/sentry/ingest/transaction_clusterer/tasks.py

```diff
@@ -91,3 +93,75 @@ def cluster_projects(projects: Sequence[Project]) -> None:
             metrics.incr(
                 "txcluster.cluster_projects.unclustered", amount=unclustered, sample_rate=1.0
             )
+
+
+@instrumented_task(
+    name="sentry.ingest.span_clusterer.tasks.spawn_span_cluster_projects",
+    queue="span.descs.clusterer",  # XXX(iker): we should use a different queue
```

Comment:
Didn't you want to use the same queue for now?

```suggestion
    queue="transactions.name_clusterer",  # XXX(iker): we should use a different queue
```

Reply:
Good catch!